settings/settings-cpt.php

```diff
@@ -131,9 +138,16 @@ protected function form() {
 						printf(
 							'<li><label><input name="taxonomies[%s]" type="checkbox" value="1" %s %s/> %s</label></li>',
 							esc_attr( $taxonomy ),
-							checked( in_array( $taxonomy, $this->options['taxonomies'] ) || $disabled, true, false ),
+							checked( $disabled || in_array( $taxonomy, $this->options['taxonomies'] ), true, false ),
 							disabled( $disabled, true, false ),
-							esc_html( $tax->labels->name )
+							esc_html(
+								sprintf(
+									/* translators: 1 is a post type or taxonomy label, 2 is a post type or taxonomy key. */
+									_x( '%1$s (%2$s)', 'taxonomy setting choice', 'polylang' ),
```

Comment:
@Chouby May think different upon his review, but I think both fields should use the same comment and context.

```suggestion
									_x( '%1$s (%2$s)', 'content type setting choice', 'polylang' ),
```